### DISCUSSION TITLE
Clase del viernes

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-EDA1#73] [Polls]
Como saben, la clase del viernes está suspendida por la ceremonia de apertura del año académico. Les propongo tener una sesión _express_ de una hora, mañana jueves (o el viernes fuera de esas horas), para hacer la revisión de las propuestas que están presentando al reto (incluyendo la revisión de mi propuesta).

**Comment by martinlopez7** [comment]
¿A qué hora es la clase hoy? Nosotros hoy tenemos clase de 12:15-14:15 y de 14:45-16:45

**Reply by mmasias** [reply]
Visto lo visto y ante la aclamación popular, quedamos a las 17:15h

**Reply by mmasias** [reply]
En la -2.6
